nautobot/ipam/templates/ipam/ipaddress_merge.html

```diff
@@ -0,0 +1,182 @@
+        <div class="col-md-10 col-md-offset-1">
+            <div class="panel panel-danger">
+                <div class="panel-heading"><strong>Confirm Merging These Duplicate IP Addresses</strong></div>
+                <div class="panel-body">
+                    <p><strong>Warning:</strong> The following operation will merge these {{ queryset.count }} IP Addresses. Please select the attributes you want to keep in the merged IP Address.</p>
+                    {% block message_extra %}{% endblock %}
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-10 col-md-offset-1">
+            <div class="panel panel-default">
+                <table class="table table-hover table-headings">
+                <thead>
+                    <th>
+                        Merging IP Addresses
+                    </th>
+                    <th>
+                        Host
+                    </th>
+                    <th>
+                        Mask Length
+                    </th>
+                    <th>
+                        Namespace
+                    </th>
+                    <th>
+                        Type
+                    </th>
+                    <th>
+                        Status
+                    </th>
+                    <th>
+                        Role
+                    </th>
+                    <th>
+                        DNS Name
+                    </th>
+                    <th>
+                        Description
+                    </th>
+                    <th>
+                        Tenant
+                    </th>
+                    <th>
+                        NAT IP (Inside)
+                    </th>
+                    <th>
+                        Tags
+                    </th>
+                </thead>
+                <tbody>
+                {% for ip in queryset %}
+                        <tr>
+                            <td>
+                                <input type="checkbox" id="pk" name="pk" value="{{ip.pk}}" checked>
+                                <label for="pk"></label>
+                            </td>
+                            <td>
+                                {% if forloop.first %}
+                                    <input type="hidden" id="host" name="host" value="{{ip.host}}" checked>
+                                {% endif %}
+                                <label for="host">{{ip.host}}</label>
+                            </td>
+                            <td>
+                                {% if forloop.first %}
+                                    <input type="radio" id="mask_length" name="mask_length" value="{{ip.mask_length}}" required checked>
+                                {% else %}
+                                    <input type="radio" id="mask_length" name="mask_length" value="{{ip.mask_length}}" required>
+                                {% endif %}
```

Comment:
Nit, but makes things a bit less repetitive:

```suggestion
                                <input type="radio" id="mask_length" name="mask_length" value="{{ip.mask_length}}" required {% if forloop.first %}checked{% endif %}>
```